middleware/proxy/upstream.go

```diff
@@ -28,13 +28,13 @@ type staticUpstream struct {
 		Path     string
 		Interval time.Duration
 	}
+	Without string
```

Comment:
The name "Without" in code is a little nebulous. Maybe something more descriptive like TrimPrefix or StripPrefix or WithoutPathPrefix or something like that. (Thoughts?)